### ISSUE TITLE
Properly handle various display rotations

### ISSUE THREAD

**Issue by godbyk** [OSVR/SteamVR-OSVR#73]
Currently SteamVR throws errors when the HDK isn't in landscape orientation.

Update the SteamVR-OSVR driver to work in portrait orientation as well.

**Comment by yangjiudan** [comment]
unlucky, my hmd is portrait. in direct display mode, the image showed on my display is rotated, only one eys shows rotated 90 degree, the other one eye is showed outside of the sceen.
when do you plan to support this feature? or can you tell me how to do it, and I can try this.

**Comment by godbyk** [comment]
@yangjiudan Preliminary support for display rotations is now in the `refactor` branch. I've only done a little testing so far, but feel free to try it out and let me know how it works for your HMD.

(If you do try it out, don't forget to init and recursively update the submodules. I moved the display detection code to a separate repository and pulled it in as a submodule.)

**Comment by godbyk** [comment]
@yangjiudan Would you mind giving the [latest release](https://bintray.com/osvr/SteamVR-OSVR/SteamVR-OSVR-Win/_latestVersion) a try? It's designed to fix problems with the display orientation.

**Comment by Balderick** [comment]
Might want to wait on feedback before closing .....

**Comment by godbyk** [comment]
@Balderick I closed this because the original issue (SteamVR-OSVR not supporting display rotation) has been resolved. My conversation with Yang can continue regardless and may end up being spun off into a separate issue if the latest release doesn't resolve the issue for him.
